src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -0,0 +1,46 @@
+import {t} from 'app/locale';
+import {NewQuery} from 'app/types';
+import {Location} from 'history';
+
+export const PERFORMANCE_EVENT_VIEW: Readonly<NewQuery> = {
+  id: undefined,
+  name: t('Performance'),
+  query: 'event.type:transaction',
+  projects: [],
+  fields: [
+    'transaction',
+    'project',
+    // TODO: rename this to rpm() once it is supported
+    'rpm',
```

Comment:
Isn't this supported now?

Reply:
@markstory this is only supported for the yAxis; it's not exposed as a column.

Reply:
Interesting. I would have thought it worked as a column too on master.

Reply:
@markstory my bad. It does work; but the alias in the payload response was the form of `rpm_1234`.

Reply:
Should be addressed.